File: patterns/c-patterns/pattern151.c
// Pattern 151: ...

// *
// *
// *
// *
// * * *

#include <stdio.h>

int main()
{
    int n = 5;
    int i, j;

    for (i = 1; i <= n; i++)
    {
        for (j = 1; j <= n; j++)
        {
            if (j == 1 || (i == n && j <= (n / 2) + 1))
            {
                printf("* ");
            }
        }
        printf("\n");
    }
    return 0;
}

File: patterns/python-patterns/pattern151.py
# Pattern 151: ...

# *
# *
# *
# *
# * * *

n = 5

for i in range(1, n + 1):
    for j in range(1, n + 1):
        if j == 1 or (i == n and j <= (n # 2) + 1):
            print("* ", end="")
    print()
